src/components/shared/RegistrationModal.tsx

```diff
@@ -236,7 +237,7 @@ const RegistrationModalContent = () => {
 							<b>
 								(<span>{t("HELP.HELP")}</span>)
 								{" "}
-								<span className="fa fa-question-circle" />
+								<LuMessageCircleQuestion />
```

Comment:
Slightly off-center:
```suggestion
<LuMessageCircleQuestion style={{ position: "relative", top: 1 }} />
```
(I'd go with `top: 1.5` honestly. A large majority of displays will be able to deal with half pixels and those that cannot will round to 1)